migrate/migrations/000042_add-generated-columns-and-indexes-to-keyvalue-table.up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE key_value
+    ADD COLUMN object VARCHAR(255)
+    GENERATED ALWAYS AS (LEFT(JSON_UNQUOTE(value->>"$.object"), 255)) STORED;
```

Comment:
`$.object` is not always a string - will the `LEFT` function handle that?

Reply:
It will output the actual object, ex: `{"id": "...", "type": "Follow", "actor": {"id": "https://..." } ..`
The `WHERE` clause will fail on that `=` comparison.